domains/mewwme.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "mewwme.is-a.dev",
+    "repo": "https://github.com/mewwme/mewwme.github.io",
+    "owner": {
+        "username": "mewwme",
```

Comment:
username needs to match your github username and you are required to add your email